webclient/app/src/app/config/DataTypes.js

```diff
@@ -0,0 +1,42 @@
+const DataTypes = [
```

Comment:
ich würde das nicht in einen config-Ordner auslagern. Ich verstehe da die Konsistenz - was kommt wohin, warum styles bei den Komponenten und das jetzt nicht - überhaubt nicht mehr.

Reply:
Das heißt jetzt in der Schnittstelle "FieldType" (in Anlehnung an jHipster) und ich denke wir sollten das auch in der UI umbenennen (in dem Falle wahrscheinlich FieldTypes).

Reply:
Extra Ticket oder hier noch?

Reply:
Wollen wir eine zusätzliche ID als Identifier mitgeben? Ist meines Erachtens nicht unbedingt notwendig. Die Datentypen-Namen werden nie gleich sein

Reply:
> Ergänzung zu gestern: Das Pattern-Feld wird bei Datentypen außer String ignoriert. Wäre also gut, wenn es da noch eine Warnung / Fehlermeldung gäbe bzw. es einfach disabled wäre.

See https://dev.azure.com/starwit/lj-projectbuilder/_sprints/backlog/lj-projectbuilder%20Team/lj-projectbuilder/v3/v3%20Sprint%204?workitem=185

Reply:
> Wollen wir eine zusätzliche ID als Identifier mitgeben? Ist meines Erachtens nicht unbedingt notwendig. Die Datentypen-Namen werden nie gleich sein

Wurde raus genommen